src/doc/book/structs.md

```diff
@@ -61,7 +61,7 @@ write something like this:
 
 ```rust,ignore
 struct Point {
-    mut x: i32,
+    mut x: i32, // this causes an error
```

Comment:
Yurk. Well, they shouldn't be. If you want you can fix them as well (in this PR or another, as you want).